src/symbol.c

```diff
@@ -34,13 +34,18 @@ static const char pack_table[] = "_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRS
 static mrb_sym
 sym_inline_pack(const char *name, uint16_t len)
 {
+  enum {
```

Comment:
I'd use `const int` here rather than `enum`. I don't want to use `enum` for a non-enumerative purpose.

Reply:
OK. I fixed it.